apps/nextjs-website/src/components/atoms/TabComponent/TabComponent.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+import React, { FC, ReactNode } from 'react';
+import { Box, Tab, Tabs, Typography } from '@mui/material';
+
+export type TabPanelProps = {
+  readonly children?: ReactNode;
+  readonly index: number;
+  readonly value: number;
+};
+
+const TabPanel: FC<TabPanelProps> = ({ children, value, index }) => {
+  return (
+    <div
+      role='tabpanel'
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+    >
+      {value === index && <Box p={3}>{children}</Box>}
+    </div>
+  );
+};
+
+export type TabItem = {
+  readonly title: string;
+  readonly content: ReactNode;
+};
+
+export type TabsProps = {
+  readonly items: ReadonlyArray<TabItem>;
+};
+
+export const TabComponent: FC<TabsProps> = ({ items }: TabsProps) => {
+  const [currentTab, setCurrentTab] = React.useState(0);
+
+  return (
+    <div>
```

Comment:
Do we need this `<div>`? Maybe a fragment is enough
```suggestion
    <>
```